feature/bgp/addpath/ate_tests/route_propagation_test/route_propagation_test.go

```diff
@@ -154,9 +154,43 @@ func TestBGP(t *testing.T) {
 		desc:     "propagate IPv4 over IPv4",
 		fullDesc: "Advertise prefixes from ATE port1, observe received prefixes at ATE port2",
 		dut: dutData{&telemetry.NetworkInstance_Protocol_Bgp{
```

Comment:
How are we checking for MRAI? (for more info https://datatracker.ietf.org/doc/html/rfc4271#section-9.2.1.1)

Following per the procedure for  RT-1.3: BGP Route Propagation

=================
Establish BGP sessions between:
ATE port-1 and DUT port-1
ATE port-2 and DUT port-2
For IPv4 and IPv6:
Advertise prefixes from ATE port-1, observe received prefixes at ATE port-2.
Specify default accept for received prefixes on DUT.
Specify table based neighbor configuration to cover - validating the supported capabilities from the DUT.
	- MRAI (minimum route advertisement interval), ensuring routes are advertised within specified time.  <<<<---
	- IPv4 routes with an IPv6 next-hop when negotiating RFC5549 - validating that routes are accepted and advertised with the specified values.
	- With ADD-PATH enabled, ensure that multiple routes are accepted from a neighbor when advertised with individual path IDs, and that these routes are advertised to ATE port-2.

Reply:
@sachendras, Do you think this might be out of scope for this PR?

Reply:
Sure